ako-operator/README.md

```diff
@@ -131,14 +131,36 @@ kubectl apply -f config/secrets/secret.yaml
 #### Deploying the AKO Controller
 
 If the AKO operator was installed using helm, a default `AKOConfig` object called `ako-config` is already added and hence, this step is not required for helm based installation.
-**Note**: If the AKO operator was installed manually, then to install the AKO controller, add an `AKOConfig` object to the `avi-system` namespace.
+**Note**: If the AKO operator was installed manually or on Openshift cluster from OperatorHub, then to install the AKO controller, add an `AKOConfig` object to the `avi-system` namespace.
```

Comment:
In the above document the name `Openshift Container Platform Web Console` and here it is `OperatorHub`. Is it ok to have both names? Which one is the actual name of web console?

Reply:
Operator is installed from OperatorHub which is accessible from Openshift Container Platform.

Reply:
got it. thanks

Reply:
I think we do not support manual installation of AKO operator using `helm`. right?

Reply:
Have commented the helm related information.